fe/free_energy.py

```diff
@@ -241,101 +136,53 @@ def _get_integrator(self, combined_masses):
             seed
         )
 
-    def vacuum_edge(self, lamb, equil_steps=10000, prod_steps=100000):
+    def prepare_vacuum_edge(self, ff_params):
         """
-        Run a vacuum decoupling simulation at a given value of lambda.
+        Prepares the vacuum system.
 
         Parameters
         ----------
-        lamb: float [0, 1]
-            0 is the fully interacting system, and 1 is the non-interacting system
-
-        equil_steps: float
-            Number of steps to run equilibration. Statistics are not gathered.
-
-        prod_steps: float
-            Number of steps to run production. Statistics are gathered.
+        ff_params: tuple of np.array
+            Exactly equal to bond_params, angle_params, proper_params, improper_params, charge_params, lj_params
 
         Returns
         -------
-        float, float
-            Returns a pair of average du_dl values for bonded and nonbonded terms.
+        4 tuple
+            unbound_potentials, system_parameters, combined_masses, combined_coords
```

Comment:
update to match return types
```suggestion
            final_potentials, final_params, combined_masses, combined_coords
```